core/workflow-core/src/main/resources/storage-config.yaml

```diff
@@ -27,6 +27,6 @@ storage:
           min-wait-ms: 100   # 0.1s
           max-wait-ms: 10000 # 10s
   jdbc:
-    url: "jdbc:postgresql://localhost:5432/texera_db"
+    url: "jdbc:postgresql://localhost:5432/texera_db?currentSchema=texera_db,public"
```

Comment:
Why do you need to make this change?

Reply:
this is equivalent to `set search_path to texera_db, public` so that the query we sent to postgres can find the `pgroonga_option` function we used in our query for fuzzy search. Otherwise, the query will throw an exception for the `pgroonga_option` function that is not found.